db/schema.rb

```diff
@@ -29,6 +29,19 @@
     t.index ["batch_id"], name: "index_allocation_batch_jobs_on_batch_id"
   end
 
+  create_table "allocation_changes", force: :cascade do |t|
+    t.bigint "school_device_allocation_id", null: false
+    t.string "category"
+    t.integer "delta"
```

Comment:
Are we keeping `delta` in the db?

Reply:
Sorry, I see `delta` is still referenced. Are we keeping the three values? `prev_allocation`, `new_allocation` and `delta`

Reply:
I wouldn´t like to have the 3 of them. I just didn´t modify it but I will do now.